client/logclient.go

```diff
@@ -35,6 +35,13 @@ type LogClient struct {
 	jsonclient.JSONClient
 }
 
+// CheckLogClient is an interface that allows (just) checking of various log contents.
+type CheckLogClient interface {
```

Comment:
Checking? or Verifying? Do we need "Log" as this is a log only package?

Reply:
It's analogous to the existing `LogClient` and `MultiLogClient` structs and the `AddLogClient` interface, so `CheckLogClient` was the best I could come up with.